cl/citations/models.py

```diff
@@ -0,0 +1,96 @@
+from django.db import models
+from eyecite.models import FullCaseCitation
+
+from cl.citations.utils import map_reporter_db_cite_type
+from cl.search.models import BaseCitation, Opinion
+
+
+class UnmatchedCitation(BaseCitation):
+    """Keep track of citations that could not be resolved to a cluster on the
+    batch citator run
+    """
+
+    UNMATCHED = 1
+    FOUND = 2
+    RESOLVED = 3
+    FAILED_AMBIGUOUS = 4
+    FAILED = 5
+    STATUS = (
+        (
+            UNMATCHED,
+            "The citation does not exist in the search_citation table."
+            " We couldn't match the citation to a cluster on the "
+            " previous citation extractor run",
+        ),
+        (
+            FOUND,
+            "The citation exists on the search_citation table. We "
+            " haven't updated the citing Opinion.html_with_citations yet",
+        ),
+        (
+            RESOLVED,
+            "The citing Opinion.html_with_citations was updated successfully",
+        ),
+        (
+            FAILED_AMBIGUOUS,
+            "The citing Opinion.html_with_citations update "
+            "failed because the citation is ambiguous",
+        ),
+        (FAILED, "The citing Opinion.html_with_citations update failed"),
+    )
+    citing_opinion: models.ForeignKey = models.ForeignKey(
+        Opinion,
+        help_text="The opinion citing this citation",
+        on_delete=models.CASCADE,
+        related_name="eyecite_citations",
```

Comment:
It's usually best to do the name of the model. I'd suggest:

```suggestion
        related_name="unmatched_citations",
```